tools/src/main/scala/scala/scalanative/codegen/CodeGen.scala

```diff
@@ -12,6 +12,13 @@ import scalanative.nir._
 
 object CodeGen {
 
+  private final class Platform(target: String) {
+    val isWindows = target.contains("indows")
```

Comment:
It could be possible, but it's actually three different things:
1. sbt os name (for sbt plugin)
2. tools has target for compilation
3. runtime method scalanative_isWindows for running application